composer.json

```diff
@@ -52,7 +52,7 @@
         "cakephp/cakephp-codesniffer": "dev-next",
         "mikey179/vfsstream": "^1.6",
         "paragonie/csp-builder": "^2.3",
-        "phpunit/phpunit": "~8.4.0"
+        "phpunit/phpunit": "~8.4.0 || ~8.5.0"
```

Comment:
Cant we use normal semver again?^^

Reply:
I thought ~ was used here because phpunit versions cannot be trusted.

Seems like we can change it to ~8.5.0 at least.

Reply:
This is one of the few times where a new minor release of phpunit didn't cause us any problems :)